sample/src/main/AndroidManifest.xml

```diff
@@ -31,17 +31,20 @@
     <activity android:name=".HelloWorldActivity"/>
     <activity android:name=".WrappedViewActivity"/>
     <activity android:name=".SwipeRefreshActivity"/>
-    <activity
-        android:name=".NavigationDrawerActivity"
-        android:theme="@style/AppTheme.NoActionBar"
-        />
-    <activity android:name=".RecyclerViewsInsideViewPagerActivity">
+    <activity android:name=".MenuActivity"
+        android:theme="@style/AppTheme.NoActionBar">
       <intent-filter>
         <action android:name="android.intent.action.MAIN"/>
 
         <category android:name="android.intent.category.LAUNCHER"/>
       </intent-filter>
     </activity>
+    <activity
+        android:name=".NavigationDrawerActivity"
+        android:theme="@style/AppTheme.NoActionBar"
+        />
+    <activity android:name=".RecyclerViewsInsideViewPagerActivity">
+    </activity>
```

Comment:
Why moving these Activities there? Is something related with this PR? If not, avoid doing it to maintain PR's cleaner, please